ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.9.yaml

```diff
@@ -60,6 +60,19 @@ base_images:
     namespace: ocp
     tag: vsphere-csi-driver-operator-test
 releases:
+  arm64-initial:
```

Comment:
this release image isn't used anywhere, yet. Am I guessing right that this is in prep for an upgrade test, for which it'll be required?

Reply:
yes, that is correct. We did initial prep in ART for future use case.